ExtraDry/ExtraDry.Server/ProblemDetailsResponse.cs

```diff
@@ -0,0 +1,48 @@
+﻿using ExtraDry.Core.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.Filters;
+using System.Diagnostics;
+using System.Net;
+
+namespace ExtraDry.Server;
+
+public static class ProblemDetailsResponse {
+
+    internal static void RewriteResponse(ExceptionContext context, HttpStatusCode code, string? details = null) => RewriteResponse(context.HttpContext, context.Exception.GetType().Name, (int)code, context.Exception.Message, details);
```

Comment:
I find this to be a bit cramped.  I'm guessing that it's functionally the same as the previous version, but it's not clear.  Getting rid of temp variables and stringing them into one-liners reduces readability (and debuggability) that I was aimin for here.

Reply:
I've spaced this out similar to the original file.